content/docs/command-reference/remote/add.md

```diff
@@ -334,6 +337,33 @@ $ dvc remote add -d myremote hdfs://user@example.com/path/to/dir
 
 <details>
 
+### Click for WebHDFS
+
+**HDFS and WebHDFS:**
+
+Both remotes, HDFS and WebHDFS, allow using a Hadoop cluster as a remote
+repository. However, HDFS relies on `pyarrow` which in turn requires `libhdfs`,
+an interface to the Java Hadoop client, that must be installed separately.
```

Comment:
Looks like this long text could be boiled down a lot (less implementation details).